includes/tables/class-wcpdf-number-store-list-table.php

```diff
@@ -193,14 +193,15 @@ public function get_search() {
 	 */
 	public function get_numbers() {
 		global $wpdb;
-
+		
+		$request                        = stripslashes_deep( $_GET );
 		$results                        = array();
-		$paged                          = $this->get_paged();
+		$paged                          = $this->get_paged( $request );
 		$offset                         = $this->per_page * ( $paged - 1 );
-		$search                         = $this->get_search();
-		$table_name                     = isset( $_GET['table_name'] ) ? sanitize_text_field( $_GET['table_name'] ) : null;
-		$order                          = isset( $_GET['order'] ) ? sanitize_text_field( $_GET['order'] ) : 'DESC';
-		$orderby                        = isset( $_GET['orderby'] ) ? sanitize_text_field( $_GET['orderby'] ) : 'id';
+		$search                         = $this->get_search( $request );
+		$table_name                     = isset( $request['table_name'] ) ? sanitize_text_field( $request['table_name'] ) : null;
```

Comment:
Do you think it's better not to take the table name from users? I don't see why we need to have this option here.

Reply:
Because you can select the table from the UI.

Reply:
Please check my last commit.